### PR TITLE
CI: Adding codestyle check for Python examples

### PR DESCRIPTION
**Pull request by thopiekar**
This is done by pycodestyle and will check against PEP8 standard.

Open questions:
 - [ ]  Shall be the same implemented to the GitLab workflow?
 - [ ]  Optionally, adding some exceptions to the check.

SET AS WIP, SINCE CHANGES WILL LIKELY BE NEEDED TO THE EXAMPLE CODE.

### CODE REVIEW

**Comment by sloriot** [comment]
Thanks, it looks good, let me know when you are done with your changes.

**Comment by thopiekar** [comment]
No problem :) I'm done.

File: .travis.yml [hunk 1]
```diff
@@ -69,6 +74,7 @@ install:
   - cd ../..
 
 script:
+  - pycodestyle --show-source --show-pep8 $PYCODESTYLE_CONVENTIONS $CGAL_PYTHON_EXAMPLES || pycodestyle --statistics $PYCODESTYLE_CONVENTIONS -qq $CGAL_PYTHON_EXAMPLES
```
Comment (sloriot) [inline .travis.yml]: could you explain what this line will do? Will we get a travis failure each time an example is not well formatted?

Comment (sloriot) [reply .travis.yml]: and more importantly is the error displayed by travis obvious enough for the developer?

Comment (thopiekar) [reply .travis.yml]: Yes and yes. You can take a look at the failed ones above.
What you get is even a full description of the coding rule and examples how it should be written instead. The second part of the line after the OR clause will just give stats about the found errors.

**Pull request merged**